src/GDGUkraine/templates/register.html

```diff
@@ -379,7 +392,7 @@
     </div>
     <div id="footer" style="text-align: center; color: rgba(50, 50, 50, 0.7);">
         <small>
-            Regform by <a href="https://google.com/+OlexandrKovalchuk">Olexandr Kovalchuk</a>, <a href="https://plus.google.com/102989169039151799851/about">Oleh Zasadnyy</a>, <a href="https://plus.google.com/112452406409308053659/about">Valentyn Shybanov</a> and <a href="https://plus.google.com/110108248841907526802/about">Svyatoslav Sydorenko</a> | <a href="https://github.com/GDG-Ukraine/gdg.org.ua/issues">Report an issue</a><br />
+            Regform by <a href="https://plus.google.com/102989169039151799851/about">Oleh Zasadnyy</a>, <a href="https://plus.google.com/112452406409308053659/about">Valentyn Shybanov</a>, <a href="https://plus.google.com/110108248841907526802/about">Svyatoslav Sydorenko</a> and <a href="https://google.com/+OlexandrKovalchuk">Oleksandr Kovalchuk</a> | <a href="https://github.com/GDG-Ukraine/gdg.org.ua/issues">Report an issue</a><br />
```

Comment:
1. I do not like my name stays first in a list like I'm the main developer, which is not true
2. It doesn't. I thought it would be handy to do it now than later.